tools/generate_hotkey.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
+
+
+general = []
+msg_actions = []
+msg_compose = []
+navigation = []
+searching = []
+stream_list = []
+
+CATEGORY = {
+    'general': general,
+    'msg_actions': msg_actions,
+    'msg_compose': msg_compose,
+    'navigation': navigation,
+    'searching': searching,
+    'stream_list': stream_list,
+}
```

Comment:
Can we just generate these from what is in `HELP_CATEGORIES`? Otherwise we'll need to update this script if we change the categories.